src/frontend/src/components/SportCard.tsx

```diff
@@ -0,0 +1,17 @@
+function SportCard(props) {
+  return (
+    <a
```

Comment:
What's the difference between `<a>` and `<A>`?

Reply:
`<A>` tags are used with the `@solidjs/router`. They have some extra properties that work with routing.